src/main/resources/templates/index.html

```diff
@@ -1,59 +1,61 @@
-<!DOCTYPE html>
-<html>
-<head>
-<meta charset="ISO-8859-1"/>
-<title>Feel Free Cincinnati</title>
-<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
-<style>
-header {color: #00FFFF; background-color: #000000;}
-h1 {font-size: 50px}
-body {background-color: #BBBBBB}
-footer {text-align: center; font-size:75%; font-style: italic}
-</style>
-</head>
-<body>
-<header><div class="text-center"><h1>Feel Free Cincinnati</h1></div>
-<div class="text-center">
-<div class="custom-control custom-radio custom-control-inline">
-  <input type="radio" id="eventRdo" name="customRadioInline1" class="custom-control-input">
-  <label class="custom-control-label" for="eventRdo">Find an Event</label>
-</div>
-<div class="custom-control custom-radio custom-control-inline">
-  <input type="radio" id="adventureRdo" name="customRadioInline1" class="custom-control-input">
-  <label class="custom-control-label" for="adventureRdo">Go on an Adventure</label>
-</div></div>
-
-</header>
-
-<br><br><br>
-<form>
-<div class="text-center">
-  <div class="form-row">
-    <div class="col">
-      <select class="custom-select" id="inputGroupSelect01"  style="width: 400px">
-     
-    <option selected style='display: none' value=''>Free Day</option>
-    <option value="1">Sunday</option>
-    <option value="2">Monday</option>
-    <option value="3">Tuesday</option>
-    <option value="3">Wednesday</option>
-    <option value="3">Thursday</option>
-    <option value="3">Friday</option>
-    <option value="3">Saturday</option>
-  </select>
-    </div>
-    <div class="col">
-      <input type="number" class="form-control" id="budget" placeholder="Budget" style="width: 400px">
-    </div>
-  </div>
-  </div> <br>
-  <p class="text-center"><button type="submit" class="btn btn-primary">Feeling Free</button></p>
-</form>
-
-
-
-<br><br><br><br><br>
-<footer>Copyright &copy; 2019 FeelFreeInc</footer>
-</body>
+<!DOCTYPE html>
```

Comment:
When reformatting, make one commit specific for the reformat, then make any other changes in a separate commit.  This makes it easier to see what changed.